judo-tatami-jsl-jsl2psm/src/test/resources/transferobject/TestCreateMappedTransferObjectTypeModel.jsl

```diff
@@ -0,0 +1,56 @@
+model MappedTransferObjectTypeModel;
+
+type string String(min-size = 0, max-size = 250);
+type numeric Integer(precision = 3, scale = 0);
+
+
+entity Entity {
+	field Integer attribute;
+	field EntityRelated containment;
+	field EntityRelated[] containmentCollection;
+	relation EntityRelated association;
+}
+
+entity EntityRelated {
+	field Integer attribute;
+	relation Entity entity opposite-add entityRelatedOpposite;
+
+}
+
+transfer UnmappedRelated {
+	field String transient;
+}
+
+transfer Mapped maps Entity as e {
+    field Integer unmappedAttribute;
+    
+    // It have to be invalid - but it's not'
+	// field UnmappedRelated unmappedContainment maps e.containment;
+	// field UnmappedRelated[] unmappedContainmentCollection maps e.containmentCollection;
+	// field UnmappedRelated unmappedContainment reads e.containment;
+	// field UnmappedRelated[] unmappedContainmentCollection reads e.containmentCollection;
```

Comment:
JIRA ticket? And/or separate negative test?

Reply:
Nope. I've reported to Privi :)